src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
```

Comment:
Note that this line is way overkill if we only use the last entry: this is really an iterator and keeping the results in memory is somewhat costly.  You could do something like:
```
event = None
for event in cres:
   pass
if event:
   ...
```